Face_Recognition/Readme.md

```diff
@@ -0,0 +1,5 @@
+## Face-Recognition
+
+This project is trained to identify the avengers like hulk, thor, captain_america and tony.
+
+The project asks the user for input and in the given photo if the above mentioned images are present then it represents the face with a rectangule with the name on it.
```

Comment:
Do correct the `rectangle` typo.
Also You can briefly describe a bit about the Model here.